dsa/DSA || GAS STATION(GREEDY) #33.py

```diff
@@ -0,0 +1,21 @@
+def gascheck(j, counter, remaining_gas, cost_for_next_station,n):
+    if (counter == n):
+        return 106
+    elif (remaining_gas >= cost_for_next_station):
+        j = j + 1
+        if (j >= n):
+            j = j - n
+        remaining_gas = gas[j] + remaining_gas - cost_for_next_station
+        cost_for_next_station = cost[j]
```

Comment:
`gas` and `cost` are also not defined inside the function

Reply:
Have u tried running the code?

Reply:
yes, but the code is working fine. even before n was taken in as an argument.